### PR TITLE
Treim Update

### PR DESCRIPTION
**Pull request by HebrewToYou**
Lots of changes...

Added new routines
Removed FURY routine
Removed mstrike_limit variable and code
Modified existing routines

### CODE REVIEW

File: scripts/treim.lic [hunk 1]
```diff
@@ -218,19 +222,12 @@ def attack_routine(attack_target)
 	if UserVars.treim[:attack_type] == "sleep"
 		waitcastrt?
 		fput "symbol of sleep"
-	elsif UserVars.treim[:attack_type] == "physical"
+	elsif UserVars.treim[:attack_type] == /physical|mstrike/
```
Comment (mrhoribu) [inline scripts/treim.lic:225]: needs =~ instead of ==

File: scripts/treim.lic [hunk 2]
```diff
@@ -242,53 +239,51 @@ def attack_routine(attack_target)
...
-		end
+		fput "attack"
 		if UserVars.treim[:stance_dance]
 			pause 1
 			waitrt?
 			fput "stance defensive"
 		end
 		return 1
-	elsif UserVars.treim[:attack_type] == "uac"
+	elsif UserVars.treim[:attack_type] == /fire|ranged/
```
Comment (mrhoribu) [inline scripts/treim.lic:249]: needs =~ instead of ==

File: scripts/treim.lic [hunk 3]
```diff
@@ -242,53 +239,51 @@ def attack_routine(attack_target)
...
+		fput "fire"
+		if UserVars.treim[:stance_dance]
 			pause 1
-			waitrt
-			fput "mstrike punch"
+			waitrt?
+			fput "stance defensive"
+		end
+		return 1
+	elsif UserVars.treim[:attack_type] == /jab|punch|grapple|kick/
```
Comment (mrhoribu) [inline scripts/treim.lic:261]: needs =~ instead of ==

File: scripts/treim.lic [hunk 4]
```diff
@@ -602,7 +502,7 @@ elsif ((variable[1] == nil || variable[1].downcase == "help") && UserVars.treim[
 	
   Running Commands
      ;send reimcount   - shows current group leader, group count, and previos group count
-     whisper ooc group clearcheck - if group members are running script,
+     whisper ooc group checkclear - if group members are running script,
```
Comment (mrhoribu) [inline scripts/treim.lic:505]: The OOC command didn't change from clearcheck that the script looks for. So may want to revert this for instructions so people don't get confused.

File: scripts/treim.lic (hunk 2 above)
Comment (HebrewToYou) [reply scripts/treim.lic]: Fixed

File: scripts/treim.lic (hunk 3 above)
Comment (HebrewToYou) [reply scripts/treim.lic]: Fixed

**Pull request merged**